components/leaflet-maps/NormalMap.vue

```diff
@@ -1,266 +1,226 @@
+                View Wiki Page
+              </a><br>
             </div>
-            <a :href="witch.link" target="_blank">More Info</a><br /><br />
+            <a :href="witch.link" target="_blank">More Info</a><br><br>
           </div>
         </div>
-      </l-popup>
+      </LPopup>
 
-      <l-icon :icon-anchor="iconAnchor">
+      <LIcon :icon-anchor="iconAnchor" class-name="">
         <div class="icon-wrapper">
           <div v-if="marker.witches.length > 1" class="icon-text">
             {{ marker.witches.length }}
           </div>
           <img :src="marker.markerIcon" class="zoomed-in-img" />
           <img class="icon-shadow" :src="shadowUrl" />
         </div>
-      </l-icon>
-    </l-marker>
-  </l-map>
+      </LIcon>
+
+    </LMarker>
+  </LMap>
 </template>
 
+
 <script>
-import getJulianDate from "assets/js/DateConversion.js";
 
-export default {
-  props: {
-    mapMarkers: {
-      type: Array,
-      required: true,
-    },
-    mapUrl: {
-      type: String,
-      required: true,
-    },
-    center: {
-      type: Array,
-      required: true,
-    },
-    zoom: {
-      type: Number,
-      requried: true,
-    },
-  },
-  data() {
-    return {
-      baseMapUrl: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
-      attribution:
-        'Map data © <a href="https://www.openstreetmap.org/copyright">OpenStreetMap contributors</a>. Historical Maps Layer, James Dorret 1750 from the <a href="https://maps.nls.uk/geo/explore/#zoom=6.6&lat=57.29330&lon=-5.04553&layers=125140579&b=1">NLS Maps API</a>',
-      locationOptions: ["residence", "detention", "placeOfDeath"],
-      locationsLabels: {
-        residence: "Residence",
-        detention: "Detention",
-        placeOfDeath: "Place of Death",
-      },
-      standardAttributes: ["sex", "occupation", "socialClass"],
-      standardAttributeLabels: {
+ export default {
+   props: {
+     mapMarkers: {
+       type: Array,
+       required: true
+     },
+     mapUrl: {
+       type: String,
+       required: true
+     },
+     center: {
+       type: Array,
+       required: true
+     },
+     zoom: {
+       type: Number,
+       requried: true
+     }
+   },
+   data () {
+     return {
+       baseMapUrl: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+       attribution: 'Map data © <a href="https://www.openstreetmap.org/copyright">OpenStreetMap contributors</a>. Historical Maps Layer, James Dorret 1750 from the <a href="https://maps.nls.uk/geo/explore/#zoom=6.6&lat=57.29330&lon=-5.04553&layers=125140579&b=1">NLS Maps API</a>',
+       locationOptions: ["residence", "detention", "placeOfDeath"],
+       locationsLabels: {
+         residence: "Residence",
+         detention: "Detention",
+         placeOfDeath: "Place of Death"
+       },
+       standardAttributes:["sex","occupation","socialClass"],
+       standardAttributeLabels:{
         sex: "Gender",
         occupation: "Occupation",
-        socialClass: "Social Class",
```

Comment:
I can't remember the rules in JS but usually, if possible, we should keep a trailing comma as it means if we add another item to the list, we don't have to add a comma to the line above and it makes our review diffs a bit cleaner

Reply:
Ruby added this back in